src/store/middleware/inputMiddleware.js

```diff
@@ -41,6 +41,16 @@ const inputMiddleware = store => next => action => {
         break;
       }
 
+      case messageReceivedFromRover.type: {
+        const { message } = action.payload;
+        if (message.type === "armIKEnabledReport") {
+          if (store.getState().input.inverseKinematics.lastSentArmIKState !== null && store.getState().input.inverseKinematics.lastSentArmIKState !== message.enabled) {
```

Comment:
This is super verbose, and no point in querying the state twice. Can we put `store.getState().input.inverseKinematics.lastSentArmIKState` into a local variable before this `if`